src/Chapter1.hs

```diff
@@ -615,7 +622,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integer -> Integer
+sumLast2 n = lastDigit + secondToLast
+  where
+    lastDigit = mod n 10
+    secondToLast = mod (div n 10) 10
```

Comment:
That's a nice solution! And there're actually several ways to implement this task! For example, you could use the `lastDigit` function from the previous task. Also, you might be interested in looking into the `divMod` function that returns the result of `div` and `mod` at the same time as a pair.